cumulusci/cli/cci.py

```diff
@@ -780,8 +781,6 @@ def task_run(config, task_name, org, o, debug, debug_before, debug_after, no_pro
             import pdb
             pdb.set_trace()
 
-        task()
-
```

Comment:
I guess you removed this for debugging and forgot to put it back?

Reply:
Oops, fixed in latest push